homework/list.js

```diff
@@ -33,16 +40,16 @@
     }
   }
 
-  function updateList(list) {
-    _listNoteContent = list;
+  list.prototype.updateList = function(list1) {
+    _listNoteContent = list1;
   }
 
-  function drawList() {
-    var list = _listNoteContent;
+  list.prototype.drawList = function() {
+    var list2 = _listNoteContent;
     var ul = document.createElement('ul');
     ul.id = 'note-title-list';
     var buff = document.createDocumentFragment();
-    list.forEach(function(note, i) {
+    list2.forEach((function(note, i) {
```

Comment:
See? A name collision that should never happen if you give your instance a better name.